cszczepaniak/cribbly-backend
# Dockerize

# What I added (link any issues addressed)
I dockerized the application (plus MySQL). You can now use the following `make` targets:
`make start`: start MySQL and the application
`make stop`: stop MySQL and the application
`make destroy`: destroy the docker-composed application. WARNING THIS WILL DELETE EVERYTHING IN YOUR DB

For `make start` to work, you need to add `./config/.env.dev`. Look at `./config/.env.example` to know what to put in it; the db password can be any random string.

# How to test it
Run `make start` (or the command it runs). See that you can hit the API on localhost port 8080. See that you can connect to MySQL on port 32000.
